lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -347,7 +367,8 @@ static int at_parse_param(const char **at_params_str,
 			while (is_lfcr(str[++i])) {
 			}
 
-			if (is_terminated(str[i]) || is_notification(str[i])) {
+			if (is_terminated(str[i]) || is_notification(str[i]) ||
+			    is_result(str + i)) {
```

Comment:
Maybe use consistent syntax `str[i]` for all conditions in the same if-statement?

Reply:
Thanks, `is_terminated` and `is_notification` take a `char`, but `is_result` takes a `char *` :)
I could use `&str[i]` if you prefer

Reply:
True, didn't thought about it... No need to change it.